client/src/layout/Header.tsx

```diff
@@ -1,13 +1,17 @@
+import './Header.scss'
+
+import { Link } from "react-router-dom"
+
 function Header () {
   return (
-    <header>
-      <a href="/">TripDesigner</a>
+    <header className="header">
+      <a className='header__title' href="/">TripDesigner</a>
 
-      <nav>
-        <ul>
-          <li>Option 1</li>
-          <li>Option 2</li>
-          <li>Option 3</li>
+      <nav className='header__nav'>
+        <ul className='header__tabs'>
+          <li className='header__tab'><Link to="/test">Create a trip</Link></li>
+          <li className='header__tab'><Link to="/my-trips">My trips</Link></li>
+          <li className='header__tab'><Link to="/settings">Setting</Link></li>
```

Comment:
I think 'settings' sounds better than 'setting', what do you think? Maybe it was just a typo, I don't know

Reply:
Good catch!